projects/components/src/load-async/wrapper/load-async-wrapper.component.ts

```diff
@@ -39,9 +40,15 @@ export class LoadAsyncWrapperComponent {
 
   public content?: TemplateRef<LoadAsyncContext>;
 
+  private readonly loaderTypeSubject: BehaviorSubject<LoaderType | undefined> = new BehaviorSubject<
+    LoaderType | undefined
+  >(undefined);
+  public readonly loaderType$: Observable<LoaderType | undefined> = this.loaderTypeSubject.asObservable();
+
   public constructor(@Inject(ASYNC_WRAPPER_PARAMETERS$) parameters$: Observable<LoadAsyncWrapperParameters>) {
     this.state$ = parameters$.pipe(
       tap(params => (this.content = params.content)),
+      tap(params => this.loaderTypeSubject.next(params.loaderType)),
```

Comment:
do we gain anything by storing it as observable? If we store the `loaderType` similar to `content`, would it cause any issues?